docs/source/markdown/podman-artifact-pull.1.md.in

```diff
@@ -0,0 +1,134 @@
+% podman-artifact-pull 1
+
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-pull - Pulls an artifact from a registry and stores it locally
+
+## SYNOPSIS
+**podman artifact pull** [*options*] *source*
+
+
+## DESCRIPTION
+podman artifact pull copies an artifact from a registry onto the local machine.
+
+
+## SOURCE
+SOURCE is the location from which the artifact image is obtained.
+
+```
+# Pull from a registry
+$ podman artifact pull quay.io/foobar/artifact:special
+```
+
+## OPTIONS
+
+[//]: # (BEGIN included file options/authfile.md)
+#### **--authfile**=*path*
+
+Path of the authentication file. Default is `${XDG_RUNTIME_DIR}/containers/auth.json` on Linux, and `$HOME/.config/containers/auth.json` on Windows/macOS.
+The file is created by **[podman login](podman-login.1.md)**. If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using **docker login**.
+
+Note: There is also the option to override the default path of the authentication file by setting the `REGISTRY_AUTH_FILE` environment variable. This can be done with **export REGISTRY_AUTH_FILE=_path_**.
+
+[//]: # (END   included file options/authfile.md)
```

Comment:
This file looks wrong, it should not be the .in file. Here there should be the proper include lines